packages/ckeditor5-fullscreen/src/fullscreenediting.ts

```diff
@@ -43,5 +45,21 @@ export default class FullscreenEditing extends Plugin {
 	 */
 	public init(): void {
 		this.editor.commands.add( 'fullscreen', new FullscreenCommand( this.editor ) );
+
+		const t = this.editor.locale.t;
+
+		// Set the Ctrl+Shift+F keystroke.
+		this.editor.keystrokes.set( 'Ctrl+Shift+F', FULLSCREEN );
```

Comment:
I see we use such convention in some places (e.g. for `bold` plugin) but it's only useful if such const can be used in multiple places. Here we can use `fullscreen` as a command name directly. Or if we want to have it as a const, then also use it in line `47` where we add the command to the editor.